legacyman_parser/parser.py

```diff
@@ -52,6 +53,10 @@ def parse_from_root():
         print("Url empty. Exiting")
         return
 
+    # Set target value based on cwd
+    if path_has_drive_component(os.getcwd()):
```

Comment:
ok, problem with the next line.  It sets the target directory as `target` in the root folder of the current drive. It should set the target as `target` in the current working directory.